Bingha/Bingha/Views/Home/Complete/CompleteViewController.swift

```diff
@@ -12,31 +12,58 @@ class CompleteViewController: UIViewController {
     //스토리보드 연결
     
     //모달 닫기 버튼
-    @IBOutlet weak var exitButton: UIImageView!
+    @IBOutlet weak var exitModalButton: UIButton!
     //상세 데이터 백그라운드 뷰
-    @IBOutlet weak var dataBackground: UIView!
+    @IBOutlet weak var dataView: UIView!
+    //빙하 레벨업 알림 뷰
+    @IBOutlet weak var icebergLevelAlertView: UIView!
     
     //~~동안 산소로 바꾸는 양
-    @IBOutlet weak var oxygenChangeAmount: UILabel!
+    @IBOutlet weak var changeToOxyLabel: UILabel!
     //탄소 배출 저감량
-    @IBOutlet weak var reducedCarbon: UILabel!
+    @IBOutlet weak var reducedCarbonLabel: UILabel!
     //오늘 총 저감량
-    @IBOutlet weak var todayReducedCarbon: UILabel!
+    @IBOutlet weak var todayReducedCarbonLabel: UILabel!
     //이동거리
-    @IBOutlet weak var moveDistance: UILabel!
+    @IBOutlet weak var moveDistanceLabel: UILabel!
     //소요시간
-    @IBOutlet weak var timeDuration: UILabel!
+    @IBOutlet weak var timeDurationLabel: UILabel!
     
     
     override func viewDidLoad() {
         super.viewDidLoad()
         
+        //View 커스텀
+        viewCornerAndShadowCustom(viewName: icebergLevelAlertView) //빙하 레벨업
+        viewCornerAndShadowCustom(viewName: dataView) //상세 데이터
+        exitButtonCustom()
+        
+        //Label 폰트 커스텀
+        
+        //데이터 연결
+        
+        
+        
```

Comment:
```suggestion
        // TODO: Label 폰트 커스텀
        // TODO: 데이터연결
```
다음과 같이 빈공백은 지양하고 TODO 주석을 활용해 보면 좋을것 같습니다.